credentials/src/main/kotlin/web5/sdk/credentials/StatusListCredential.kt

```diff
@@ -68,7 +73,7 @@ public object StatusListCredential {
    * val statusListCredential = StatusListCredential.create("http://example.com/statuslistcred/id123", "http://example.com/issuers/1", StatusPurpose.REVOCATION, listOf(vc1,vc2))
    * ```
    */
-  @Throws(RuntimeException::class)
+  @Throws(StatusListCredentialCreateException::class)
```

Comment:
Does the `@Throws` annotation automatically add info to the docs? Or do we need to add a `* @throws` comment in the lines above?

Reply:
It doesn't appear that it does. From https://tbd54566975.github.io/web5-kt/docs/htmlMultiModule/credentials/web5.sdk.credentials/-status-list-credential/create.html I see what's shown below.

![image](https://github.com/TBD54566975/web5-kt/assets/1139998/1140d688-0358-417a-af38-d732fa98d119)

Reply:
added `@throws` in ktdocs and `@Throws` annotations

Reply:
Great candidate for another linting rule. Added https://github.com/TBD54566975/tbd-detekt-rules/issues/7